lib/internal/modules/cjs/loader.js

```diff
@@ -1120,6 +1120,14 @@ Module._preloadModules = function(requests) {
     parent.require(requests[n]);
 };
 
+Module.syncESMExports = function syncESMExports() {
```

Comment:
I'm wondering if it might make sense to note this is for _builtins_... `syncBuiltinESMExports`? Don't want to bike shed / make more complex, but trying to think about readability.

Reply:
i have no strong opinion, can update

Reply:
👍 for adding "Builtin" to the method name. Maybe even `syncBuiltinExports`?